ntbs-service/Services/LegacySearchService.cs

```diff
@@ -167,6 +170,7 @@ private async Task<NotificationBannerModel> AsNotificationBannerAsync(dynamic re
                 TbService = tbService?.Name,
                 TbServiceCode = tbService?.Code,
                 TbServicePHECCode = tbService?.PHECCode,
+                CaseManager = result.CaseManagerGivenName != null ? $"{result.CaseManagerGivenName} {result.CaseManagerFamilyName}" : null,
```

Comment:
Could it ever be the case that the given name is null, but the family name isn't?

I also wonder if it's worth pulling this out into a separate method.

Reply:
Fair enough, it might be worth adding a comment to that effect here